### DISCUSSION TITLE
Will chia also support a decentralized cloud space?

### DISCUSSION THREAD

**Discussion by PatrickGlatt** [Chia-Network/chia-blockchain#2466] [Ideas]
Hi, 
what’s about using chia also as a decentralized cloud? I mean, I am filling my HDDs with tons of  100GB files, but nobody can use this space. All this PB of space are lost just for farming. Why not using it as cloud space and maybe even sell this space in XCH or other Cryptos. 
Idk if this is even possible but this would make chia much more interesting/green and less waste of space^^

**Comment by dorkmo** [comment]
yes that is an issue with chia. unfortunately chia is set up to use the space for confirmation of farming. there are a few other crypto coins that try to enable the use of the space for cloud use, but chia has no plans to do this.
